org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/util/IdCache.xtend

```diff
@@ -16,18 +16,21 @@
 
 package org.eclipse.sprotty.util
 
-import com.google.common.collect.BiMap
-import com.google.common.collect.HashBiMap
+import java.util.Map
 import java.util.Set
 import org.eclipse.sprotty.SModelElement
+import org.apache.log4j.Logger
 
 /** 
  * Helps to create unique IDs for {@link SModelElement}s.
  */
 class IdCache <T> {
 	
-	val BiMap<String, T> id2element = HashBiMap.create
-	val Set<String> otherIds = newHashSet
+	static val LOG = Logger.getLogger(IdCache)
+
+	Map<T, String> element2id = newHashMap
+	Map<String, T> id2element = newHashMap
+	Set<String> otherIds = newHashSet
```

Comment:
It seems you have reverted my changes https://github.com/eclipse/sprotty-server/commit/c5d5b1418526c14cae73d6de848aae8558128e8a#diff-792329e50cbbdd3a4b69713fb7e205fb
Was this on purpose?